### DISCUSSION TITLE
Fine-Tune BERT Transformer with spaCy 3

### DISCUSSION THREAD

**Discussion by CarlaMPR** [explosion/spaCy#11262] [Help: Other Questions] [answered]
Hello! I'm trying to train a custom NER model by fine-tuning a transformer model with spaCy 3.  In the base. config.cfg file:
**[components.transformer.model]
@architectures = "spacy-transformers.TransformerModel.v3"
name = "roberta-base"
tokenizer_config = {"use_fast": true}**

what models can I use on "name" instead of roberta-base? For example, can I change to this one https://huggingface.co/dmis-lab/biobert-v1.1 ?

Thank you!!

**Comment by rmitsch** [comment, marked as answer]
Hi @CarlaMPR, see the [docs on transformers in spaCy](https://spacy.io/api/transformer):
> It supports all models that are available via the [HuggingFace transformers](https://huggingface.co/transformers) library.

`biobert` should work as well. If you run into any problems, let us know here.

**Reply by CarlaMPR** [reply]
Thank You! So, as biobert doesn't appear on this page, https://huggingface.co/docs/transformers/index#supported-models, does it mean I can't use it?

**Reply by rmitsch** [reply]
As a rule of thumb: as long as the model can be loaded with [`AutoModel.from_pretrained()`](https://huggingface.co/transformers/v3.0.2/model_doc/auto.html), it should work (we can't guarantee that though). So go ahead and give it a shot.

I recommend to have a look at the [spaCy + HuggingFaceFAQ](https://github.com/explosion/spaCy/discussions/10768) for other potential useful info.

**Reply by CarlaMPR** [reply]
Thank you!
